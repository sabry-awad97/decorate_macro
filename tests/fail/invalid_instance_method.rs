use decorate_macro::decorate;

struct Test {
    value: i32,
}

impl Test {
    #[decorate(self.invalid..method)] // Invalid syntax
    fn test(&self) -> i32 {
        self.value
    }
}

fn main() {}
