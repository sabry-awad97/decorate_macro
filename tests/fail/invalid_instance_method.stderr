error: expected `,`
 --> tests/fail/invalid_instance_method.rs:8:20
  |
8 |     #[decorate(self.invalid..method)] // Invalid syntax
  |                    ^
